app/models/scc_account.rb

```diff
@@ -268,7 +268,7 @@ def revalidate_subscription_status(elements, scc_id)
   # params: items_to_invalidate: ActiveRecord_(*)
   #         save_record: store in database or not (default)
   # return: ActiveRecord elements with invalidated subscription status
-  def invalidate_subscription_status(items_to_invalidate, save_record = false)
+  def invalidate_subscription_status(items_to_invalidate, save_record: false)
```

Comment:
In line 175 and 210, the parameter is now specified, even though the value for `save_record` equals the default-value.
Is this something rubocop requested?

As far as I can see the parameter is now specified in all method-calls, so it is technically no longer necessary to set a default-value for it :thinking:

Reply:
Good point. It is the `Style/OptionalBooleanParameter` cop. I think this is related to readability. They say that if you specify a boolean parameter, you can also write an own method in most cases. But if you need it, it makes sense to specify the parameter in the method call such that you understand what it happening inside the function call (instead of just passing `true` or `false` as parameter, then you most likely don't know what is going on). 

With this being said, yes, I can probably remove the default value and keep it as a keyword argument

Reply:
I get the readability argument an it makes sense :thinking: 
Another way to solve this is to add wrapper-functions that have the set-flags in their name.

But we can also fix that later :smile:

Reply:
Not sure if it makes sense for this function. It is just one line that triggers saving.